pipelines/deploy.py

```diff
@@ -41,26 +40,35 @@ def get_deploys():
     Returns:
         Deploy list.
     """
+    res = []
     client = init_pipeline_client()
-    runs = []
-    token = ''
-    while True:
-        list_runs = client.list_runs(
-            page_token=token, sort_by='created_at desc', page_size=100)
-        if list_runs.runs is not None:
-            for run in list_runs.runs:
-                # check if run is type deployment
-                manifest = run.pipeline_spec.workflow_manifest
-                if 'SeldonDeployment' in manifest:
-                    _run = format_pipeline_run(run)
-                    runs.append(_run)
-            token = list_runs.next_page_token
-            runs_size = len(list_runs.runs)
-            if runs_size == 0 or token is None:
-                break
-        else:
-            break
-    return {'runs': runs}
+    runs = client.list_runs(sort_by='created_at').runs
+
+    # Get cluster Ip
+    try:
+        config.load_incluster_config()
+        v1 = client.CoreV1Api()
+        service = v1.read_namespaced_service(
+            name='istio-ingressgateway', namespace='istio-system')
+        ip = service.status.load_balancer.ingress[0].ip
+    except:
+        ip = ""
+
+    if runs:
```

Comment:
Tava pensando aqui...essa API está olhando para os runs, não para os SeldonDeployments, certo?
Parece conceitualmente estranho... Depois quando for fazer undeploy teria que excluir o run senão continuaria aparecendo na lista...

Faz sentido o que eu falei? Dá pra trocar o list_runs por um "list_SeldonDeployment"?